.buildkite/pipeline.yml

```diff
@@ -35,7 +35,7 @@ steps:
   - label: ":lint-roller: loop device cleanup"
     agents:
       queue: "${BUILDKITE_AGENT_META_DATA_QUEUE:-default}"
-      distro: "${BUILDKITE_AGENT_META_DATA_DISTRO}"
+      distro: "debian"
```

Comment:
Can you revert this?  We should be able to run all of the exact same steps on both distros without a different file.